app/src/main/java/com/example/myfridgehome/MainActivity.kt

```diff
@@ -109,6 +52,8 @@ open class MainActivity : AppCompatActivity() {
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
 
+
+        // Maybe refactor to single service and inject services?
```

Comment:
Don't have a conversation in source code comments.   This is clutter, which is technical debt.  What I recommend: since you are in your own branch, you are working in a sandbox, separate from master.  You can propose your changes in code, commit and push them to your own branch, and then describe the rationale behind the changes in the pull request.